security_monkey/watchers/sqs.py

```diff
@@ -76,27 +76,28 @@ def slurp(self):
                         continue
 
                     try:
-                        policy = self.wrap_aws_rate_limited_call(
+                        attrs = self.wrap_aws_rate_limited_call(
                             q.get_attributes,
-                            attributes='Policy'
+                            attributes='All'
                         )
-                        if 'Policy' in policy:
-                            try:
-                                arn = 'arn:aws:sqs:{region}:{account_number}:{name}'.format(
-                                    region=region.name,
-                                    account_number=account_number,
-                                    name=q.name)
-
-                                json_str = policy['Policy']
-                                policy = json.loads(json_str)
-                                policy['arn'] = arn
-
-                                item = SQSItem(region=region.name, account=account, name=q.name, arn=arn,
-                                               config=policy)
-                                item_list.append(item)
-                            except:
-                                self.slurp_exception((self.index, account, region, q.name), InvalidAWSJSON(json_str),
-                                                     exception_map, source="{}-watcher".format(self.index))
+                        try:
+                            if 'Policy' in attrs:
+                                json_str = attrs['Policy']
+                                attrs['Policy'] = json.loads(json_str)
+                            else:
+                                attrs['Policy'] = {}
+
+                            # Delete ephemeral values
+                            del attrs['ApproximateNumberOfMessagesNotVisible']
+                            del attrs['ApproximateNumberOfMessages']
+                            del attrs['ApproximateNumberOfMessagesDelayed']
```

Comment:
Instead of deleting ephemeral things, we can just tell SM that they are ephemeral:

Example from S3: https://github.com/Netflix/security_monkey/blob/develop/security_monkey/watchers/s3.py#L15

Example from IAM User: https://github.com/Netflix/security_monkey/blob/develop/security_monkey/watchers/iam/iam_user.py#L14

Just put this in the watcher constructor:
```python
self.ephemeral_paths = [
    'ApproximateNumberOfMessagesNotVisible',
    'ApproximateNumberOfMessages',
    'ApproximateNumberOfMessagesDelayed']
```